French translation for ressource manager

JeanPaulGontier opened the pull request:
@microsoft-github-policy-service agree

JeanPaulGontier committed:
French translation for ressource manager

JeanPaulGontier commented:
@microsoft-github-policy-service agree

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.fr-FR.resx:
```diff
@@ -0,0 +1,168 @@
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="ControlTitle.Text" xml:space="preserve">
+    <value>{0} type de dossier</value>
```
```suggestion
    <value>Type de dossier {0}</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.fr-FR.resx:
```diff
@@ -0,0 +1,168 @@
+    <value>{0} type de dossier</value>
+  </data>
+  <data name="ControlTitle_editfoldermapping.Text" xml:space="preserve">
+    <value>Modifier la représentation du dossier</value>
```
```suggestion
    <value>Éditer le mappage de dossier</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.fr-FR.resx:
```diff
@@ -0,0 +1,168 @@
+    <value>Définir un nouveau type de dossier.</value>
+  </data>
+  <data name="DuplicateMappingName.Text" xml:space="preserve">
+    <value>Une représentation avec le même nom existe. Les cangements n'ont pas été enregistrés.</value>
```
```suggestion
    <value>Un mappage avec le même nom existe déjà. Les changements n'ont pas été enregistrés.</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.fr-FR.resx:
```diff
@@ -0,0 +1,168 @@
+    <value>Une représentation avec le même nom existe. Les cangements n'ont pas été enregistrés.</value>
+  </data>
+  <data name="FolderProviderSettings.Text" xml:space="preserve">
+    <value>Réglages fournisseur de dossier.</value>
```
```suggestion
    <value>Réglages du fournisseur de dossier.</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.fr-FR.resx:
```diff
@@ -0,0 +1,168 @@
+    <value>Fournisseur de dossier.</value>
+  </data>
+  <data name="NameLabel.Help" xml:space="preserve">
+    <value>Entrez le nom de la représentation du dossier.</value>
```
```suggestion
    <value>Entrez le nom du mappage du dossier.</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.fr-FR.resx:
```diff
@@ -0,0 +1,168 @@
+    <value>Nom</value>
+  </data>
+  <data name="RequiredFields.Text" xml:space="preserve">
+    <value>Indique champ requis.</value>
```
```suggestion
    <value>Indique les champs requis.</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.fr-FR.resx:
```diff
@@ -0,0 +1,168 @@
+    <value>Indique champ requis.</value>
+  </data>
+  <data name="SelectFolderProvider.Text" xml:space="preserve">
+    <value>- Selection fournisseur de dossier -</value>
```
```suggestion
    <value>- Selectionnez un fournisseur de dossier -</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>Le fichier que vous voulez charger existe déjà dans le dossier.</value>
+  </data>
+  <data name="FileUploadedMessage.Text" xml:space="preserve">
+    <value>Fichier chargé.</value>
```
```suggestion
    <value>Fichier téléversé.</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>Fichier chargé.</value>
+  </data>
+  <data name="FileUploadStoppedMessage.Text" xml:space="preserve">
+    <value>Chargement du fichier stoppé</value>
```
```suggestion
    <value>Chargement du fichier arrêté</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>URL</value>
+  </data>
+  <data name="ItemNameRequiredMessage.Text" xml:space="preserve">
+    <value>Nom requis</value>
```
```suggestion
    <value>Le nom requis</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>Nom de l'élément</value>
+  </data>
+  <data name="LastModifiedOnDate.Text" xml:space="preserve">
+    <value>Date dernière modification</value>
```
```suggestion
    <value>Date de dernière modification</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>Le fichier icone de groupe ne peut pas être effacé.</value>
+  </data>
+  <data name="InvalidExtensionMessage.Text" xml:space="preserve">
+    <value> type de fichier non autorisé pour chargement</value>
```
```suggestion
    <value> type de fichier non autorisé pour téléversement</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>Rafraîchir</value>
+  </data>
+  <data name="Sync.Text" xml:space="preserve">
+    <value>Synchro</value>
```
```suggestion
    <value>Synchroniser</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>Synchro</value>
+  </data>
+  <data name="SyncThisFolder.Text" xml:space="preserve">
+    <value>Synchro ce dossier</value>
```
```suggestion
    <value>Synchroniser ce dossier</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>Synchro ce dossier</value>
+  </data>
+  <data name="SyncThisFolderAndSubfolders.Text" xml:space="preserve">
+    <value>Synchro ce dossier &amp; sous dossiers</value>
```
```suggestion
    <value>Synchroniser ce dossier &amp; sous dossiers</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>Fournisseur de dossier</value>
+  </data>
+  <data name="FolderTypeDefinitions.Text" xml:space="preserve">
+    <value>Définition type de dossier</value>
```
```suggestion
    <value>Définitions de types de dossiers</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>Déplacer élément(s)</value>
+  </data>
+  <data name="NewLocation.Text" xml:space="preserve">
+    <value>Nouvelle localisation</value>
```
```suggestion
    <value>Nouvel emplacement</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>Eléments</value>
+  </data>
+  <data name="MappedPath.Text" xml:space="preserve">
+    <value>Chemin de représentation</value>
```
```suggestion
    <value>Chemin du mappage</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>Télécharger</value>
+  </data>
+  <data name="Upload.Text" xml:space="preserve">
+    <value>Charger</value>
```
```suggestion
    <value>Téléverser</value>
```

JeanPaulGontier replied:
normalement on utilise le terme téléverser en Français, sauf que dans les autres traductions dnn c'est charger qui est utilisé, es ce qu'il ne vaut pas mieux utiliser la même traduction partout ?

valadas replied:
Hmm, je préfères vraiment le mot téléverser car is est plus précis, mais en effet le reste dit charger...

Je suis correct avec l'un ou l'autre mais je préfère téléverser même si on doit éventuellement ajuster le reste.

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>Remplacer</value>
+  </data>
+  <data name="EditFolderMappings.Text" xml:space="preserve">
+    <value>Modifier la réprésentation du dossier</value>
```
```suggestion
    <value>Modifier le mappage du dossier</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/View.ascx.fr-FR.resx:
```diff
@@ -0,0 +1,129 @@
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="EditModule.Text" xml:space="preserve">
+    <value>Modifier Module</value>
```
```suggestion
    <value>Éditer Module</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/View.ascx.fr-FR.resx:
```diff
@@ -0,0 +1,129 @@
+    <value>Modifier Module</value>
+  </data>
+  <data name="GroupModeError.Text" xml:space="preserve">
+    <value>Le module est en mode groupe, mais la page non</value>
```
```suggestion
    <value>Le module est en mode groupe, mais la page ne l'est pas</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/View.ascx.fr-FR.resx:
```diff
@@ -0,0 +1,129 @@
+    <value>Le module est en mode groupe, mais la page non</value>
+  </data>
+  <data name="UserModeError.Text" xml:space="preserve">
+    <value>Le module est en mode user, mais l'utilisateur n'est pas identifié</value>
```
```suggestion
    <value>Le module est en mode utilisateur, mais l'utilisateur n'est pas identifié</value>
```

valadas reviewed (changes requested):
Thanks, I made some comments to better match the rest of the terms in other places and types, etc.

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

attention "mappage" n'est pas un mot français, mais c'est peut être judicieux de l'utiliser car mapping n'a pas de traduction directe dans ce contexte

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/View.ascx.fr-FR.resx

effectivement, c'est mieux d'utiliser éditer comme partout ailleurs dans dnn

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/View.ascx.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/View.ascx.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

soit il faut mettre nom requis soit le nom est requis mais pas le nom requis

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>Fichiers globaux.</value>
+  </data>
+  <data name="GroupAssets.Text" xml:space="preserve">
+    <value>{0} fichiers</value>
```
```suggestion
    <value>Fichiers pour {0}</value>
```

valadas commented on Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx:
```diff
@@ -0,0 +1,441 @@
+    <value>Fichiers du site</value>
+  </data>
+  <data name="UserAssets.Text" xml:space="preserve">
+    <value>{0} Fichiers</value>
```
```suggestion
    <value>Fichiers pour {0}</value>
```

valadas reviewed (changes requested):
Noticed another little thing, in context those are

MyAwesomeGroup files => Fichiers pour MyAwesemeGroup
John Smith files => Fichiers pur John Smith

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

JeanPaulGontier committed:
Update Resources/DesktopModules/ResourceManager/App_LocalResources/ResourceManager.fr-FR.resx

Co-authored-by: Daniel Valadas <info@danielvaladas.com>

valadas reviewed (approved):
Excellent, thank you / merci !
